templates/gmetad.conf.erb

```diff
@@ -102,12 +102,16 @@ gridname "<%= scope.lookupvar('::ganglia::gmetad::gridname') %>"
 # is always trusted.
 # default: There is no default value
 # trusted_hosts 127.0.0.1 169.229.50.165 my.gmetad.org
+trusted_hosts <%= @trusted_hosts.join(' ') %>
```

Comment:
Joshua Hoblitt <notifications@github.com> writes:

>> @@ -102,12 +102,16 @@ gridname "<%= scope.lookupvar('::ganglia::gmetad::gridname') %>"
>>  # is always trusted.
>>  # default: There is no default value
>>  # trusted_hosts 127.0.0.1 169.229.50.165 my.gmetad.org
>> +trusted_hosts <%= @trusted_hosts.join(' ') %>

> This causes failures under the future parser (which are allowed to fail
> in the travis matrix due to a change in erb `:undef` handing) -- could
> you add a condition check for nil and check the output of the future
> parser runs?

        Okay, I think I have it working the way you'd expect: 

    https://github.com/jhoblitt/puppet-ganglia/pull/48

        The CI build passes, the only errors associated with
Jenkins appear to be associated with 'gridname' in gmetad.conf and
'override_hostname' in gmond.conf (neither of which I touched).

                           - Tim Skirvin (tskirvin@killfile.org)
-- 
http://wiki.killfile.org/           Skirv's Homepage <FISH>< <*>
http://wiki.killfile.org/software/              Skirv's Software